### PR TITLE
Enhance bootstrap shell script

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

To make the error message more clear when `SPARK_HOME` is invalid.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: bin/load-kyuubi-env.sh [hunk 1]
```diff
@@ -76,7 +76,7 @@ if [[ -f ${KYUUBI_HOME}/RELEASE ]]; then
   SPARK_BUILTIN="${KYUUBI_HOME}/externals/spark-$SPARK_VERSION_BUILD-bin-hadoop${HADOOP_VERSION_BUILD:0:3}"
 fi
 
-if [[ ! -d ${SPARK_BUILTIN} ]]; then
+if [[ ! -f ${KYUUBI_HOME}/RELEASE && ! -d ${SPARK_BUILTIN} ]]; then
```
Comment (yaooqinn) [inline bin/load-kyuubi-env.sh:79]: just `else` here is enough

Comment (yaooqinn) [reply bin/load-kyuubi-env.sh:79]:
```
-fi
-
-if [[ ! -d ${SPARK_BUILTIN} ]]; then
+else
```

**Comment by yaooqinn** [comment]
```shell
if [[ -z ${SPARK_HOME} ]]; then
  echo "Error: SPARK_HOME IS NOT SET! CANNOT PROCEED." >&2
  exit 1
elif [ ! -d ${SPARK_HOME} ]; then
  echo "Error: SPARK_HOME[${SPARK_HOME}] DOES NOT EXIST! CANNOT PROCEED." >&2
  exit 1
else
  if [[ ! -x "${SPARK_HOME}/bin/spark-submit" ]]; then
    echo "Error: INVALID SPARK DISTRIBUTION! CANNOT PROCEED." >&2
    exit 1
  fi
fi
```

add a new branch for `bin/kyuubi`

File: bin/load-kyuubi-env.sh (hunk 1 above)
Comment (pan3793) [reply bin/load-kyuubi-env.sh]: Updated.

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks ! merged to master and 1.3

**Pull request closed without merging**